highfive/newpr.py

```diff
@@ -124,13 +124,21 @@ def set_assignee(self, assignee, owner, repo, issue, user, author, to_mention):
                 client.send_then_quit("{}: ping to review issue https://www.github.com/{}/{}/pull/{} by {}."
                     .format(irc_name_of_reviewer, owner, repo, issue, author))
 
+        commands = {}
         if to_mention and len(to_mention) > 0:
             message = ''
             for mention in to_mention:
                 if len(message) > 0:
                     message += '\n\n'
                 message += "%s\n\ncc %s" % (mention['message'],
                                             ','.join([x for x in mention['reviewers'] if x != user]))
+                cmd = mention.get('command')
```

Comment:
Here.

As far as I understood it, the PR allows to add something like `"command": "@bot do-action"` to mentions, and you want to use that to run UI tests whenever rustdoc is changed. You also added a new mention for rustdoc, but there is no trace of `"command": "..."`.

Reply:
Oh I see. Good catch!